src/test/java/org/mockitousage/matchers/CapturingArgumentsTest.java

```diff
@@ -315,4 +315,17 @@ public void captures_correctly_when_captor_used_on_pure_vararg_method() throws E
         verify(mock).varargs(eq(42), argumentCaptor.capture());
         Assertions.assertThat(argumentCaptor.getValue()).contains("capturedValue");
     }
+
+    @Test
+    public void should_capture_null_vararg() {
+        // given
+        ArgumentCaptor<String> argumentCaptor = ArgumentCaptor.forClass(String.class);
+
+        // when
+        mock.varargs(null);
```

Comment:
Hi @ChristianSchwarz 

I will add that test and let see what happens. Should work :)